src/mixxx.cpp

```diff
@@ -451,7 +469,15 @@ void MixxxMainWindow::finalize() {
     Timer t("MixxxMainWindow::~finalize");
     t.start();
 
-    // Save the current window state (position, maximized, etc)
+    UserSettingsPointer pConfig = m_pSettingsManager->settings();
+    int inhibit = pConfig->getValue<int>(ConfigKey("[Config]","InhibitScreensaver"));
+    mixxx::ScreenSaverPreference inhiPref = mixxx::ScreenSaverPreference(inhibit);
```

Comment:
In this case we could also avoid casting to enum and cast to int below. 
The line is not aware the new PREVENT_ON_PLAY feature. 

So it may look finally like that: 
`if (inhiPref != static_cast<int>(mixxx::ScreenSaverPreference::PREVENT_OFF)) {`